doc/plugin_server_bundlepublisher_aws_s3.md

```diff
@@ -13,7 +13,7 @@ The plugin accepts the following configuration options:
 | bucket            | The Amazon S3 bucket name to which the trust bundle is uploaded.                                                                                               | Yes.                                                                   |                                                     |
 | object_key        | The object key inside the bucket.                                                                                                                              | Yes.                                                                   |                                                     |
 | format            | Format in which the trust bundle is stored, &lt;spiffe &vert; jwks &vert; pem&gt;. See [Supported bundle formats](#supported-bundle-formats) for more details. | Yes.                                                                   |                                                     |
-
+| endpoint          | A custom S3 endpoint should be set when using third-party object storage providers, such as Minio.                                                             | No.                 |                                                     |
```

Comment:
The `Default` column contains the default value for the configuration. Since there's no default here, could you leave it blank?

Could you also align the column markers with the previous rows? I doesn't matter for the rendered markdown, but it makes it a bit easier when editing the files.